osquery/tables/system/windows/certificates.cpp

```diff
@@ -57,8 +57,17 @@ typedef struct _ENUM_ARG {
 } ENUM_ARG, *PENUM_ARG;
 
 template <typename Iterator>
-inline void toHexStr(Iterator begin, Iterator end, std::string& output) {
-  boost::algorithm::hex(std::string(begin, end), back_inserter(output));
+inline void toHexStr(Iterator begin,
```

Comment:
Is there an easy way to add a test for this behavior?

Reply:
My test was to use CertMgr (the UI) and manually compare the results, but for automating this:
- I suppose a test could query the cert store for a Microsoft root cert and check that the serial number is the expected one?
- Or it could insert a known test cert to the user cert store and read that back?

Reply:
Ah, I was more concerned with the correctness of `toHexStr`, testing the BE and LE modes.